### PR TITLE
Update conf for congestion control algorithm

### PR DESCRIPTION
**Pull request by lee-emqx**

### CODE REVIEW

**Review by wanghaEMQ — APPROVED** [review]

**Review by JaylinYu — APPROVED** [review]

File: CMakeLists.txt [hunk 1]
```diff
@@ -177,6 +177,7 @@ endif()
 if(NNG_ENABLE_QUIC)
   set(BUILD_QUIC_CLI ON)
   add_definitions(-DSUPP_QUIC)
+  add_definitions(-DQUIC_API_ENABLE_PREVIEW_FEATURES)
```
Comment (JaylinYu) [inline CMakeLists.txt:180]: what is this for?

File: etc/nanomq.conf [hunk 2]
```diff
@@ -124,7 +124,7 @@ bridge.mqtt {
 	nodes [
 		{
 			name="emqx"
-			address="mqtt-tcp://127.0.0.1:1883"
+			address="mqtt-tcp://localhost:1883"
```
Comment (JaylinYu) [inline etc/nanomq.conf:127]: better not

**Review by JaylinYu — CHANGES_REQUESTED** [review]
need to switch back to original conf file

**Comment by JaylinYu** [comment]
sorry I have to close this due to merging difficulities

**Pull request closed without merging**